docs/recipes/react.md

```diff
@@ -0,0 +1,98 @@
+# Testing React components
+
+This recipe is giving guidelines to test React components.
```

Comment:
This doesn't really have much value. It's just a longer description of the title. I would drop it or come up with something else.